ruby/Gemfile

```diff
@@ -0,0 +1,8 @@
+ruby '>=2.5.0'
```

Comment:
is it necessary add in this file?
```
source 'https://rubygems.org'
```
I can't compile without *source*.
Or can you share another way for compilation, please?

Reply:
It's is normal syntax for bundler Gemfile. 'source' just tells bundler where to find gems.

Reply:
Maybe add it here or represent in README, that it is neccessary.
I think, that code must compile by tutorial without hidden appends in configuration.